test/unit/event.js

```diff
@@ -2876,7 +2867,7 @@ QUnit[ jQuery.fn.click ? "test" : "skip" ]( "Event aliases", function( assert )
 	} );
 } );
 
-// Support: IE9 (remove when IE9 is no longer supported)
+// Support: IE9 only
```

Comment:
So [this](https://github.com/jquery/contribute.jquery.org/issues/95) is not landed, still open and yet we already using it as standard :/